docs/lit/examples/01-overview.jl

```diff
@@ -38,11 +40,51 @@ using InteractiveUtils: versioninfo
 # ### Overview
 
 #=
-todo
+Although neural networks are very popular,
+low-dimensional nonlinear regression problems
+can be handled quite efficiently
+by kernel regression (KR).
```

Comment:
Should we say "kernel ridge regression (KRR)" (and use KRR instead of KR elsewhere) to be consistent with `PERK.krr` and `PERK.krr_train`?

Reply:
Yes - done.